### PR TITLE
docs: add missing word

### PR DESCRIPTION
**Pull request by tamas-nemeth**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

word 'to' missing

Issue Number: N/A

## What is the new behavior?

grammar fixed

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/deployment.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ Check out [angular-cli-ghpages](https://github.com/angular-buch/angular-cli-ghpa
 
 ## Server configuration
 
-This section covers changes you may have make to the server or to files deployed to the server.
+This section covers changes you may have to make to the server or to files deployed to the server.
```
Comment (santoshyadav198613) [inline aio/content/guide/deployment.md]: This section covers changes you may have to make to the server or to files deployed on the server.

**Review by santoshyadavdev — COMMENTED** [review]
LGTM

**Review by kapunahelewong — APPROVED** [review]
LGTM. Thank you, @tamas-nemeth!

Could you rebase? Here are my own workflow notes if you'd like them as a reference. Let me know if you have any questions.

**Comment by jbogarthyde** [comment]
@tamas-nemeth Could you pull from master and try rebasing again? I can't seem to restart the failed test (which looks like it has nothing to do with the PR content).

**Comment by jbogarthyde** [comment]
This is a very small docs-only change that should not require further approval. Test failures don't appear relevant.

**Comment by jbogarthyde** [comment]
CARETAKER: This seems to be failing tests for reasons unrelated to the content. It is a small doc-only change that should not need further approval.

**Pull request closed without merging**